packages/pn-commons/src/components/CustomTooltip.tsx

```diff
@@ -31,7 +31,7 @@ function CustomTooltip({ openOnClick, tooltipContent, children, sx, onOpen }: Pr
       <Box sx={sx}>
         <Tooltip
           arrow
-          leaveTouchDelay={0} // this makes the tooltip not disappearing after some seconds by click (useful for mobile view)
+          leaveTouchDelay={5000}
```

Comment:
@NicolaGiornetta why 5000? how does the tooltip behaves in mobile view?

Reply:
Because of "dischargeable" tooltip required by accessibility. In mobile, the tooltip will disappear after 5000 seconds. I did not found anything about timing in accessibility, so I set as 5 seconds.